gtests/net/mptcp/add_addr/add_addr4_server.pkt

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
in the commit description, we should maybe avoid `script` for the .pkt file: I was looking for a shell script :stuck_out_tongue:

Reply:
> in the commit description, we should maybe avoid `script` for the .pkt file: I was looking for a shell script

indeed, packetdrill is a script interpreter (the [google project page](https://github.com/google/packetdrill)  defines the program as a "scripting tool")